### PR TITLE
This replaces the Ibex markdown issues templates with YAML issues tem…

### PR DESCRIPTION
**Pull request by DBees**
…plate based on CVA6

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: CVE2 bug
+description: Create a CVE2 bug
+title: "[BUG] <title>"
+labels: ["Type:Bug"]
+body:
+- type: checkboxes
```
Comment (MikeOpenHWGroup) [inline .github/ISSUE_TEMPLATE/bug.yaml:6]: Do we want to continue with these "checkboxes"?  I am not sure they add a lot of value and a good percentage of Contributors simply ignore them.

File: .github/ISSUE_TEMPLATE/task.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: CVE2 Task
+description: Create a CVE2 Project Task
+title: "[TASK] <title>"
+labels: ["task"]
+body:
+- type: checkboxes
```
Comment (MikeOpenHWGroup) [inline .github/ISSUE_TEMPLATE/task.yaml:6]: Same comment re. checkboxes

**Comment by MikeOpenHWGroup** [comment]
Hi @DBees, can you reply to my comments re. the checkboxes?

(Also, it is considered good form to have short titles for issues/pull-requests.  The "long story" can go in the comment field.

**Comment by christian-herber-nxp** [comment]
> Hi @DBees, can you reply to my comments re. the checkboxes?
> 
> (Also, it is considered good form to have short titles for issues/pull-requests. The "long story" can go in the comment field.

can you please align this? Template is still ibex

**Review by MikeOpenHWGroup — APPROVED** [review]

**Comment by MikeOpenHWGroup** [comment]
Thanks @DBees.  Approving and merging.

**Pull request merged**